games/index.js

```diff
@@ -1,23 +1,25 @@
+import games from './nodeindex';
+
 export default {
   connect4: {
-    name: 'Connect 4',
+    name: games.connect4.name,
```

Comment:
You might be able to replace all these with ```...games.connect4``` so that each property doesn't have to be explicitly copied over

Reply:
I tried this first and it didn't work for some reason. Did we change which parser we're using at some point?

Reply:
Huh. I tried it locally (node v9.2.0) and it worked, but I tried it just now on the server (node v8.2.1) and it didn't. I think this might be new in Node.js 9. Feel free to keep it as it is for now, probably safer.